packages/browser/src/transport/startFlagEvaluationBatch.ts

```diff
@@ -0,0 +1,34 @@
+import type { PageMayExitEvent, RawError } from '@datadog/browser-core'
+import {
+  createBatch,
+  createFlushController,
+  createHttpRequest,
+  createIdentityEncoder,
+  Observable,
+} from '@datadog/browser-core'
+import type { FlaggingConfiguration } from '../domain/configuration'
+
+export function startFlagEvaluationBatch(
```

Comment:
Why do we need this additional method + file? Could we put this logic directly in `createBatch`?